### PR TITLE
1640 simplify log download

### LINKED ISSUE #1640: Simplify agent log download
**Is your feature request related to a problem? Please describe.**
Users always add island log and never add monkey logs, even though monkey logs are more important they are harder to find. IF you press on logs, you won't see monkey logs to download: 
image

It's also not clear what log will be downloaded from the map, the island or the monkey? 
image

**Describe the solution you'd like**
Log pages should show monkey logs as well, map should specify which logs you can download. Island node should show 2 logs to download: 1 islands' and 1 agents'.

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 

Fixes #1640 

- Streamlines log download procedure by making all logs downloadable from a single page in UI
- Renames "Logs" page to "Telemetries", because it actually displays telemetries, not logs

image
image
image

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Have you successfully tested your changes locally? Elaborate:
    > Tested by downloading monkey logs from island
    > Tested by downloading island logs from island
    > Tested by downloading monkey logs from another machine
* [x] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - "Communicate as Backdoor User" PBA's HTTP requests to request headers only and
   include a timeout. #1577
 - The setup procedure for custom server_config.json files to be simpler. #1576
+- All logs can be downloaded from the Map page. #1640
```
Comment (mssalvatore) [inline CHANGELOG.md:16]: Maybe this should go in `Added`? 

```
### Added
...
- The ability to download the Monkey Island logs from the Infection Map page. #1640
```

File: docs/content/FAQ/_index.md [hunk 2]
```diff
@@ -156,13 +156,25 @@ The Monkey performs queries out to the Internet on two separate occasions:
...
 
-You can download the Monkey Island's log file directly from the UI. Click the "log" section and choose **Download Monkey Island internal logfile**, like so:
+Logs can be found in the Map page. Click on the machine you want to download logs from
+and press "Download log" button on the side panel.
 
-![How to download Monkey Island internal log file](/images/faq/download_log_monkey_island.png "How to download Monkey Island internal log file")
+Island server logs are downloadable from island machine.
 
-It can also be found as a local file on the Monkey Island server system in the specified
+Agent logs are downloadable from any exploited machine (where agents ran).
```
Comment (mssalvatore) [inline docs/content/FAQ/_index.md:166]:
```suggestion
Both Monkey Agent and Monkey Island logs can be found in the Map page. Click on the machine you want to download logs from and press the "Download log" button on the side panel. Note that you can only download the Monkey Island log by clicking on the Monkey Island machine in the infection map.
```

File: docs/content/FAQ/_index.md [hunk 3]
```diff
@@ -156,13 +156,25 @@ The Monkey performs queries out to the Internet on two separate occasions:
...
+
+![How to download logs](/images/faq/log_download.jpg "How to download logs")
+
+### Log locations
+
+If logs can't be downloaded through the UI,
+you can extract log files directly from the machine.
```
Comment (mssalvatore) [inline docs/content/FAQ/_index.md:173]:
```suggestion
If the logs can't be downloaded through the UI for any reason,
you can collect the log files directly from the machine where and Agent or Monkey Island ran.
```

File: docs/content/FAQ/_index.md [hunk 4]
```diff
@@ -156,13 +156,25 @@ The Monkey performs queries out to the Internet on two separate occasions:
...
+
+#### Monkey Island server log
+
+Island's log file is located in the
```
Comment (mssalvatore) [inline docs/content/FAQ/_index.md:177]:
```suggestion
The Monkey Island's log file is located in the
```

**Review by mssalvatore — CHANGES_REQUESTED** [review]

**Review by shreyamalviya — APPROVED** [review]

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**